translations/binaries/slps_strings_items_armor.json

```
@@ -929,392 +929,392 @@
             "string": "道化の名を冠した男性用衣装。[LINE]\nその昔、笑いで世界を救おうと[LINE]\n旅をした道化人の衣装。[END]",
             "original_bytes_length": 81,
             "available_bytes_size": 87,
-            "translation": "광대의 이름을 딴 남성용 의상입니다.[LINE]\n한때, 웃음으로 세상을 구하려고[LINE]\n여행했던 광대의 의상입니다.[END]"
+            "translation": "광대라는 이름이 붙은 의상.[LINE]\n먼 옛날, 웃음으로 세상을 구하려고[LINE]\n여행했던 광대가 입었다.[END]"
         },
         {
             "offset": 1786952,
             "string": "[Sign_Shield]ヴァーミリオン[END]",
             "original_bytes_length": 20,
             "available_bytes_size": 23,
-            "translation": "[Sign_Shield]버미리온[END]"
+            "translation": "[Sign_Shield]버밀리온[END]"
         },
         {
             "offset": 1786880,
             "string": "美しい朱色の紋章入りの衣装。[LINE]\n朱色が血のようにも見える[LINE]\n怪しげな服。[END]",
             "original_bytes_length": 67,
             "available_bytes_size": 71,
-            "translation": "아름다운 주황색 문장이 있는 의상입니다.[LINE]\n주황색이 피처럼 보이는[LINE]\n수상한 옷입니다.[END]"
+            "translation": "문양이 새겨진 아름다운 붉은색 의상.[LINE]\n붉은색이 피처럼 보여서[LINE]\n어쩐지 수상하다.[END]"
         },
         {
             "offset": 1786856,
             "string": "[Sign_Shield]エグゼキュータント[END]",
             "original_bytes_length": 24,
             "available_bytes_size": 24,
-            "translation": "[Sign_Shield]엑제큐턴트[END]"
+            "translation": "[Sign_Shield]엑시큐던트[END]"
         },
         {
             "offset": 1786760,
             "string": "演奏者の名を冠する男性用衣装。[LINE]\n演奏者の邪魔にならないように[LINE]\n衣擦れの音はまったくしない。[END]",
             "original_bytes_length": 89,
             "available_bytes_size": 95,
-            "translation": "연주자의 이름을 딴 남성용 의상입니다.[LINE]\n연주자의 방해가 되지 않도록[LINE]\n옷이 스치는 소리는 전혀 나지 않습니다.[END]"
+            "translation": "연주자라는 이름이 붙은 의상.[LINE]\n연주자에게 방해가 되지 않도록[LINE]\n옷이 스치는 소리는 나지 않는다.[END]"
         },
         {
             "offset": 1786744,
             "string": "[Sign_Shield]ミスウェア[END]",
             "original_bytes_length": 16,
             "available_bytes_size": 16,
-            "translation": "[Sign_Shield]미스웨어[END]"
+            "translation": "[Sign_Shield]미스 웨어[END]"
         },
         {
             "offset": 1786648,
             "string": "「伝説」を意味する男性用衣装。[LINE]\n身に纏った者は英雄と謳われる。[LINE]\n選ばれた者のみが手にできる。[END]",
             "original_bytes_length": 91,
             "available_bytes_size": 95,
-            "translation": "‘전설’을 의미하는 남성용 의상입니다.[LINE]\n몸에 걸친 자는 영웅으로 불립니다.[LINE]\n선택된 자만이 손에 넣을 수 있습니다.[END]"
+            "translation": "「전설」을 의미하는 의상.[LINE]\n걸친 자는 영웅이라고 칭송받게 된다.[LINE]\n선택받은 자만이 손에 넣을 수 있다.[END]"
         },
         {
             "offset": 1786624,
             "string": "[Sign_Shield]レザークローク[END]",
             "original_bytes_length": 20,
             "available_bytes_size": 23,
-            "translation": "[Sign_Shield]레더 망토[END]"
+            "translation": "[Sign_Shield]레더 클라우크[END]"
```

```suggestion
            "translation": "[Sign_Shield]레더 클로크[END]"
```

https://github.com/ToD-DC-Kor/kor-patch/pull/443#issuecomment-2358602133

> クローク가 데스티니2에서 "클라우크" 로 번역되어 있음 + 기존 망토 아이템 일본명 マント 분류로 따로 있어서 일괄적으로 클라우크로 처리해뒀는데 혹시 아니다 싶으면 수정 부탁드려요-

"클라우크"라니.. 요즘에는 납득하기 힘든 번역명이군요. 🤔. 지금 아이템들을 전부 영어 표기를 음차하고 있으니 Cloak를 음차한 "클로크" 정도가 어떨까 싶습니다.